lang/src/org/partiql/lang/eval/visitors/SelectListItemAliasVisitorTransform.kt

```diff
@@ -0,0 +1,63 @@
+package org.partiql.lang.eval.visitors
```

Comment:
Note that all of the new `VistiorTransform` implementations live in a different package than the `AstRewriterBase` implementations.